src/clients/UBAClient/UBAClientUtilities.ts

```diff
@@ -401,19 +401,21 @@ async function getFlows(
     return validWithinBounds || hasMatchingDeposit;
   });
 
-  const refundRequests: UbaFlow[] = spokePoolClient.getRefundRequests(fromBlock, toBlock).filter((refundRequest) => {
-    const result = refundRequestIsValid(chainId, chainIdIndices, spokePoolClients, hubPoolClient, refundRequest);
-    if (!result.valid && logger !== undefined) {
-      logger.info({
-        at: "UBAClient::getFlows",
-        message: `Excluding RefundRequest on chain ${chainId}`,
-        reason: result.reason,
-        refundRequest,
-      });
-    }
+  const refundRequests: UbaFlow[] = spokePoolClient
+    .getRefundRequests(fromBlock, toBlock)
+    .filter(async (refundRequest) => {
```

Comment:
You may want to use `filterAsync` in `ArrayUtils.ts`

Reply:
Implemented: https://github.com/across-protocol/sdk-v2/pull/272/commits/61e56bd591669ae429af2d179f7dd589b0c9b025